frb_rust/src/rust2dart.rs

```diff
@@ -74,7 +76,11 @@ impl TaskCallback {
     }
 
     /// Create a new [StreamSink] of the specified type.
-    pub fn stream_sink<T: IntoDart>(&self) -> StreamSink<T> {
+    pub fn stream_sink<T, D>(&self) -> StreamSink<T>
+    where
+        T: IntoIntoDart<D>,
+        D: IntoDart,
+    {
         StreamSink::new(self.rust2dart.clone())
```

Comment:
nit: is `D` used here or not? If not maybe we can remove it, then it is even easier for the generated code

Reply:
The `D` is needed as generic argument for the `IntoIntoDart` trait on `T`.
We could remove it if we allow any `T`. That is probably fine because `StreamSink` also allows every `T`. I just wanted to constrain it to detect errors in the generated code easier

Reply:
I see, looks reasonable! Then it may be helpful to briefly add a comment why we do this